app/src/components/features/mocksV2/MockEditorIndex/utils.ts

```diff
@@ -48,3 +49,68 @@ export const getEditorLanguage = (fileType?: FileType) => {
       return "json";
   }
 };
+
+export const validateHeaders = (headers: { [key: string]: string }) => {
+  const headerArray = Object.values(headers).map((item) => {
+    if (item && typeof item === "object") {
+      //@ts-ignore
+      return { name: item.name || "", value: item.value || "" };
+    } else {
+      return { name: "", value: "" };
+    }
+  });
+
+  const HeaderErrors = [];
+  const seenHeaderNames = new Set();
+
+  for (let i = 0; i < headerArray.length; i++) {
+    const header = headerArray[i];
+    const error: { typeOfError?: string; errorIndex?: number } = {};
+
+    if (!header.name) {
+      error.typeOfError = "name Header name is required";
```

Comment:
Set type of error to either `name` or `value`, and add other property to error object named `description` which will be rendered below respective inputs.

Reply:
Also remove the error type suffix added on each error message